src/pages/ReimbursementAccount/BusinessInfo/substeps/AddressBusiness.tsx

```diff
@@ -31,10 +31,10 @@ const INPUT_KEYS = {
     zipCode: companyBusinessInfoKey.ZIP_CODE,
 };
 
-const REQUIRED_FIELDS = [companyBusinessInfoKey.STREET, companyBusinessInfoKey.CITY, companyBusinessInfoKey.STATE, companyBusinessInfoKey.ZIP_CODE];
+const STEP_FIELDS = [companyBusinessInfoKey.STREET, companyBusinessInfoKey.CITY, companyBusinessInfoKey.STATE, companyBusinessInfoKey.ZIP_CODE];
```

Comment:
so are we keeping the screaming case or camel case?

Reply:
For constant stuff should be used UPPER_CASE as  I know, when we define it out of the component body.  Some variables was defined as a camelCase, I don't know why. and I think there is no defined rule for that in contribution guide.